tests/input/examples/personinfo_basic/transform/personinfo-to-agent.transform.yaml

```diff
@@ -25,7 +25,27 @@ class_derivations:
         populated_from: name
 
       age:
-        expr: "str({age_in_years})+' years'"
+        expr: "str({age_in_years}) + ' years'"
+
+      driving_since:
+        expr: |
+            d_test = [x.important_event_date for x in src.has_important_life_events if str(x.event_name) == "PASSED_DRIVING_TEST"]
+            target = d_test[0] if d_test else None
```

Comment:
this is an example of one of the simpler types of transform that I will need (there are more complex conversions, too)

Reply:
I'll see about getting the python compiler to treat `expr`s like this in an appropriate manner